### PR TITLE
fix(vue-renderless): [cascader] fix cascade component multiple selection configuration emitPath: false does not echo the selected name

### PR DESCRIPTION
**Pull request by falcon-jin**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #843 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by yoyo201626** [comment]
测试场景
```vue
<template>
  <tiny-cascader v-model="value" :options="options" :props="{ emitPath: false, multiple: true }"></tiny-cascader>
</template>

<script setup>
import { ref } from 'vue'
import { Cascader as TinyCascader } from '@opentiny/vue'

const value = ref('')
const options = ref([
  {
    value: 'zhinan',
    label: '指南',
    children: [
      {
        value: 'anzhuang',
        label: '安装',
        children: [
          {
            value: 'xiangmudengji',
            label: '项目登记'
          },
          {
            value: 'huanjingzhunbei',
            label: '环境准备'
          },
          {
            value: 'anzhuangcli',
            label: '安装 CLI'
          },
          {
            value: 'chuangjianxiangmu',
            label: '创建项目'
          }
        ]
      }
    ]
  }
])
</script>
```
输入框不会回显：
image

File: packages/renderless/src/cascader-panel/index.ts [hunk 1]
```diff
@@ -328,8 +328,12 @@ export const getCheckedNodes =
         return flag
       }
       for (let i = 0; i < checkedValue.length; i++) {
-        if (Array.isArray(checkedValue[i]) && checkedValue[i].length) {
-          flag = checkedValue[i][checkedValue[i].length - 1] === str
+        if (Array.isArray(checkedValue[i]) ) {
+          if(checkedValue[i].length){
```
Comment (yoyo201626) [inline packages/renderless/src/cascader-panel/index.ts:332]: 建议简化
```JavaScript
flag = checkedValue[i].length && checkedValue[i][checkedValue[i].length - 1] === str;
```

**Review by yoyo201626 — CHANGES_REQUESTED** [review]
bug分析：
`checkedValue`在`emitPath=false && multiple=true`的情况下，是一个一维数组。
而原本的代码中，只考虑了二维数组的情况（`emitPath=false && multiple=true`），导致查找不到选中值

File: packages/renderless/src/cascader-panel/index.ts [hunk 2]
```diff
@@ -328,8 +328,12 @@ export const getCheckedNodes =
         return flag
       }
       for (let i = 0; i < checkedValue.length; i++) {
-        if (Array.isArray(checkedValue[i]) && checkedValue[i].length) {
-          flag = checkedValue[i][checkedValue[i].length - 1] === str
+        if (Array.isArray(checkedValue[i]) ) {
```
Comment (yoyo201626) [inline packages/renderless/src/cascader-panel/index.ts:331]: if语句后面多了个空格

**Review by yoyo201626 — CHANGES_REQUESTED** [review]

**Review by yoyo201626 — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Comment by kagol** [comment]
@all-contributors please add @falcon-jin  for code

**Comment by kagol** [comment]
@falcon-jin 恭喜你成为OpenTiny 2023年度开源新星，请添加下OpenTiny小助手微信：opentiny-official，如已添加，请发一条信息给小助手，谢谢！

**Pull request merged**